WordPress/Tests/Commenting/NoReturnVoidUnitTest.inc

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Return types should not be void.
+ *
+ * @return void
```

Comment:
For thoroughness, it would be good to add some tests that also include a comment:

```
 * @return void Nothing.
```